docs/api/touch-bar-button.md

```diff
@@ -2,7 +2,7 @@
 
 > Create a button in the touch bar for native macOS applications
 
-Process: [Main](../tutorial/application-architecture.md#main-and-renderer-processes)
+Process: Main
```

Comment:
@bandantonio After checking some other documents, I think the correct way should be:

```
Process: [Main](../glossary.md#main-process)
```

Reply:
Thank you, didn't notice it right away